.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -56,8 +56,8 @@
 -->
 
 - This PR fixes or closes issue: fixes #
-- This PR is related to issue: 
-- Link to documentation pull request: 
+- This PR is related to issue:
+- Link to documentation pull request:
```

Comment:
Those spaces are here to ensure people can just copy-paste things in and works... not really needed, but 🤷

Reply:
Yeah, I like that space. Wish there was a better way to avoid autoformatting it away. I'll add it back.

Reply:
First I have to find the setting in vscode to turn the formattng off. :dizzy_face: